tests/System.Data.SQLite.Tests/TestUtility.cs

```diff
@@ -49,5 +49,16 @@ static void SetupCommand(IDbCommand command, string commandText, object paramete
 				}
 			}
 		}
+
+		public static void ForceLoggingInitialization()
+		{
+			// access this property so that sqlite3_config is called, which needs to happen before any other sqlite3_* method calls
```

Comment:
This is only necessary because of the `SubscribeUnsubscribeLog` test, i.e. because there is a test that uses a log handler? Might be good to mention that here.

Reply:
Correct. It has to be called before any `SQLite3Connection` code is run, so that attaching the log handler works later (in that test).